mslib/msui/_tests/test_mscolab.py

```diff
@@ -344,6 +401,21 @@ def test_close_help_dialog(self):
         self.window.close()
         assert self.window.help_dialog is None
 
+    @mock.patch("PyQt5.QtWidgets.QMessageBox")
+    @mock.patch("sys.exit")
+    def test_create_dir_exceptions(self, mockexit, mockbox):
+        with mock.patch("fs.open_fs", new=ExceptionMock(fs.errors.CreateFailed).raise_exc):
+            self.window.data_dir = "://"
+            self.window.create_dir()
+            assert mockbox.critical.call_count == 1
+            assert mockexit.call_count == 1
+
+        with mock.patch("fs.open_fs", new=ExceptionMock(fs.opener.errors.UnsupportedProtocol).raise_exc):
+            self.window.data_dir = "://"
+            self.window.create_dir()
```

Comment:
self.main_window ?

Reply:
I have fixed this issue. I'll push the commit by today evening.